lib/team_api/cross_referencer.rb

```diff
@@ -140,10 +141,20 @@ def self.create_xref_data(site)
     private_class_method :create_xref_data
 
     def self.xref_tags_and_team_members(site, tag_categories, team_xref)
+      team = (site.data['team'] || {})
       tag_categories.each do |category|
-        xrefs = create_tag_xrefs(site, (site.data['team'] || {}).values,
-          category, team_xref)
-        site.data[category] = xrefs unless xrefs.empty?
+        xrefs = create_tag_xrefs(site, team.values, category, team_xref)
+        next if xrefs.empty?
```

Comment:
It avoids a block with indentation and an extra `end` line. I also find `next if empty?` to be perfectly readable as English.